README.md

```diff
@@ -1,95 +1,26 @@
-# FortuneCookies
+# Fortune Cookie Generator
 
-## Run the Model
+### Prerequisites
 
-* `docker-compose up`
+* [docker](https://www.docker.com/products/docker)
+* [docker-compose](https://docs.docker.com/compose/)
 
-## Machine Learning
+### Run the Latest Model
 
-* Refamiliarize ourselves with our current code
-* Improve our model
-  * Add word embedding?
-  * Param Training
-  * Data collection/refine trainset
-  * Rank results / see improvement in the model
+1. Execute `docker-compose up` in your terminal.
+2. [Open the jupyter notebook](https://github.com/jupyter/docker-stacks#quick-start) in your browser.
 
-## Infrastructure
+### Goal
 
-* Create a docker image/docker compose
-* Continuous Integration
-* Continuous Deployment
+Use a [recurrent neural network](https://towardsdatascience.com/illustrated-guide-to-recurrent-neural-networks-79e5eb8049c9) to produce new fortune cookie fortunes. This repo shows three different attempts to produce a working model to produce new fortunes.
 
-## App
+### Progress Thus Far
 
-* Render model results as plain text on an html page
-* Build flask prototype
-* Get a domain
-* Website-to-model 
-* Feedback from within UI
+* Attempt One: GRU with corpus embedding. Largely based on this blog post. TODO: Add blog post link.
```

Comment:
here ya go! https://www.kaggle.com/shivamb/beginners-guide-to-text-generation-using-lstms